pkg/apis/core/types.go

```diff
@@ -2533,6 +2548,27 @@ const (
 	PodQOSBestEffort PodQOSClass = "BestEffort"
 )
 
+// PodDNSParams defines the DNS parameters of a pod.
+type PodDNSParams struct {
+	// A list of DNS name server IP addresses.
+	// +optional
+	Nameservers []string
+	// A list of DNS search domains for host-name lookup.
+	// +optional
+	Searches []string
+	// A list of DNS resolver options.
```

Comment:
Document that this is merged, not appended to the base values, from the DNSPolicy

Reply:
Done.